src/main/java/uk/gov/justice/digital/hmpps/csr/api/domain/ShiftType.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.csr.api.domain
+
+import java.util.*
+
+enum class ShiftType(val shiftType: String, val number: Int) {
+    SHIFT("SHIFT", 0),
+    OVERTIME("OVERTIME", 1);
+
```

Comment:
As above with this ENUM.
Remind me to talk to you about task_shift and task_overtime. I think you can work out what they're for but still...
https://github.com/ministryofjustice/cmd-api/blob/main/src/main/java/uk/gov/justice/digital/hmpps/cmd/api/domain/ShiftNotificationType.kt